src/docker/Dockerfile

```diff
@@ -0,0 +1,129 @@
+# permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT
+# NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND
+# FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+# THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+# INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+# SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
+# HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED
+# OF THE POSSIBILITY OF SUCH DAMAGE.
+
+FROM ubuntu:14.04
+MAINTAINER Yegor Bugayenko <yegor@teamed.io>
+LABEL Description="This is the image for aether.jcabi.com" Vendor="jcabi.com" Version="1.0"
+WORKDIR /tmp
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+# UTF-8 locale
+RUN locale-gen en_US en_US.UTF-8
+RUN dpkg-reconfigure locales
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US.UTF-8
+
+# Basic Linux tools
+RUN apt-get update
+RUN apt-get install -y wget bcrypt curl
+RUN apt-get install -y unzip zip
+RUN apt-get install -y gnupg gnupg2
+RUN apt-get install -y jq
+RUN apt-get install -y bsdmainutils
+RUN apt-get install -y libxml2-utils
+RUN apt-get install -y build-essential
+RUN apt-get install -y automake autoconf
+
+# Git 2.0
+RUN apt-get install -y software-properties-common python-software-properties
+RUN add-apt-repository ppa:git-core/ppa
+RUN apt-get update
+RUN apt-get install -y git git-core
+
+# SSH Daemon
+RUN apt-get install -y ssh && \
+  mkdir /var/run/sshd && \
+  chmod 0755 /var/run/sshd
+
+# Ruby
+RUN apt-get install -y libmagic-dev zlib1g-dev
+RUN apt-add-repository ppa:brightbox/ruby-ng
+RUN apt-get update
+RUN apt-get install -y ruby2.2 ruby2.2-dev
+RUN gem update && gem install nokogiri && gem install bundler
+
+# Ruby via RVM
+#RUN apt-add-repository -y ppa:rael-gc/rvm
+#RUN apt-get update
+#RUN sudo apt-get install -y rvm
+#RUN /bin/bash -l -c "rvm install ruby"
+#RUN /bin/bash -l -c "gem update"
+#RUN /bin/bash -l -c "gem install nokogiri"
+#RUN /bin/bash -l -c "gem install bundler"
+#RUN echo 'source /usr/share/rvm/scripts/rvm' >> /etc/bash.bashrc
+
+# PHP
+RUN apt-get install -y php5 php5-dev php-pear
+RUN curl --silent --show-error https://getcomposer.org/installer | php
+RUN mv composer.phar /usr/local/bin/composer
+RUN mkdir jsl && \
+  wget --quiet http://www.javascriptlint.com/download/jsl-0.3.0-src.tar.gz && \
+  tar xzf jsl-0.3.0-src.tar.gz && \
+  cd jsl-0.3.0/src && \
+  make -f Makefile.ref && \
+  mv Linux_All_DBG.OBJ/jsl /usr/local/bin && \
+  cd .. && \
+  rm -rf jsl
+# RUN pecl install xdebug-beta && \
+#   echo "zend_extension=xdebug.so" > /etc/php5/cli/conf.d/xdebug.ini
+
+# Java
+RUN apt-get install -y default-jdk
```

Comment:
@mkordas According to this http://packages.ubuntu.com/trusty/default-jdk , the default jdk of ubuntu 14.04 is 1.7-51 , but personally I have no guarantee. The author said to reuse Dockerfile from rultor tag 1.61.3, so that's what I did. 

Do you think another specific jdk should be used?